model/setup.sql

```diff
@@ -0,0 +1,105 @@
+CREATE SCHEMA tournamentbuzz;
+USE tournamentbuzz;
+
+CREATE TABLE users (
+  email VARCHAR(255) NOT NULL UNIQUE CHECK(email REGEXP '^[A-Za-z0-9][A-Za-z0-9]*@[A-Za-z0-9][A-Za-z0-9]*\\.[A-Za-z0-9]*$'),
+  passw VARCHAR(255) NOT NULL CHECK(passw LIKE '%________%'),
+  uname VARCHAR(60),
+  sadmin BOOL DEFAULT FALSE NOT NULL, 
+  PRIMARY KEY(email)
+);
+
+CREATE TABLE tournaments (
+	id INT(10) NOT NULL UNIQUE AUTO_INCREMENT,
+    creator VARCHAR(255) NOT NULL,
+    sponsor VARCHAR(255) DEFAULT NULL,
+    teamEvent BOOL NOT NULL DEFAULT FALSE,
+    location VARCHAR(255) DEFAULT NULL,
+    scoringType ENUM('Points') NOT NULL DEFAULT 'Points',
+    tname VARCHAR(255) DEFAULT NULL,
+    bracketType ENUM('Single Elim', 'Double Elim', 'Round-robin') NOT NULL DEFAULT 'Single Elim',
+    entryCost INT(5) NOT NULL DEFAULT 0,
+    maxParticipants INT(5) NOT NULL DEFAULT 16,
+    startDate DATE DEFAULT NULL,
+    endDate DATE DEFAULT NULL,
+    PRIMARY KEY(id),
+    FOREIGN KEY(creator)
+    REFERENCES users(email)
+);
+
+CREATE TABLE matches (
+	id INT(12) NOT NULL UNIQUE AUTO_INCREMENT,
+    location VARCHAR(255) DEFAULT NULL,
+    score VARCHAR(255) DEFAULT NULL,
+    mtime DATETIME DEFAULT NULL,
+    mname VARCHAR(255) DEFAULT NULL,
```

Comment:
Weird name, why not "name" or "matchName"? Usually I think we can make the columns table-agnostic, so that we use the table name to identify what that column would mean.